app/portal/chapters/[id]/page.js

```diff
@@ -40,8 +40,9 @@ const showEditButton = () => {
 }
 
 const columns = [
-  { field: "memberName", headerName: "Name", width: 250 },
-  { field: "emailAddress", headerName: "Email", width: 300 },
+  { field: "id", headerName: "ID", width: 120 },
```

Comment:
Did we need this id field column? It doesn't add value for the end user and might cause confusion with out of order id values.

Reply:
We definitely don't! I just removed this line! 😄